Chart.yaml

```diff
@@ -1,8 +1,9 @@
 ---
 apiVersion: v2
 name: zabbix
-version: 3.1.0
-appVersion: 6.2.0
+version: 3.1.1
+# LTS Zabbix version by default due to stability. See: https://www.zabbix.com/life_cycle_and_release_policy
+appVersion: 6.0.0
```

Comment:
here we should have the latest LTS: 6.0.7

Reply:
I agree. Doing...

Reply:
@sa-ChristianAnton using the .Chart.appVersion don't works for Zabbix tags:
https://hub.docker.com/r/zabbix/zabbix-server-mysql/tags?page=1&name=6.0.7

In this case, the name of tag has distroname + version number. 

The .Chart.appVersion not support distroname. I will remove the .Chart.appVersion of templates. Do you agree?

Reply:
Too bad, didn't think about this. Sorry. Back to before then....

Reply:
Technically the concatenation of the names is good challenge, but I think that cause the confusing for other people when they need  change the version and this aggregate few value for helm chart. Do you agree?

Reply:
I think you may be right. let's then go back to how it originally was.

Reply:
Thank you for your understanding. Doing...